modules/component-store/spec/component-store.spec.ts

```diff
@@ -1500,6 +1501,60 @@ describe('Component Store', () => {
       expect(projectorExecutionCount).toBe(3);
     });
 
+    it('uses default equality function when equality function is not provided', () => {
```

Comment:
Do we need a test for passing a custom equality check?

Reply:
Yes, we already have tests for a custom equality function.